devito/ir/support/basic.py

```diff
@@ -23,6 +22,11 @@ class IndexMode(Tag):
 REGULAR = IndexMode('regular')
 IRREGULAR = IndexMode('irregular')
 
+Delta = Symbol(name='Δ')
```

Comment:
I would refrain from using symbols with FD meaning as they may be used to define Function/....

Reply:
OK replacing with ⋈